kubernetes/websocket/wsclient.c

```diff
@@ -0,0 +1,331 @@
+#include "wsclient.h"
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <unistd.h>
+#include <fcntl.h>
+#include <stdbool.h>
+
+#define WSC_ATTACH_STDIN_BUFFER_SIZE 1024
+#define TTY_STDIN_NUMBER 0
+#define TTY_STDOUT_NUMBER 1
+#define WS_PROTOCOL_DELIM "://"
+#define WS_BASE_PATH_DELIM ":"  /* ip:port */
```

Comment:
It can't clarify the port in IPV6 address
for example, if address is [fd04::1]:443
the port is not distinguished.

Reply:
It's a problem in ipv6 env.
But the delim ":" is right even in the format `[fd04::1]:443`.
The fix should use `strrchr` in the function of `get_server_address_and_port`.
I will do it.

Reply:
Done.